internal/provisioner/equality/equality.go

```diff
@@ -157,6 +157,11 @@ func LoadBalancerServiceChanged(current, expected *corev1.Service) (*corev1.Serv
 		changed = true
 	}
 
+	if !apiequality.Semantic.DeepEqual(current.Spec.IPFamilyPolicy, expected.Spec.IPFamilyPolicy) {
+		updated.Spec.IPFamilyPolicy = expected.Spec.IPFamilyPolicy
+		changed = true
+	}
```

Comment:
I believe we'll want this change for `ClusterIPServiceChanged` and `NodePortServiceChanged` as well.